app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,7 @@
       </script>
     <% end %>
     <title>
-      <%= get_title_for_environment %>
+      <%= page_title %>
```

Comment:
Ah interesting. We just said `Simple Dashboard` everywhere, and now this method lets us define additional content per-page. Toit.

Reply:
👍